recipes/klayout/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "0.27.12" %}
+
+package:
+  name: klayout
+  version: {{ version }}
+
+source:
+  url: https://github.com/KLayout/klayout/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ef5e6ae5ea387618663949a1efa105406aeb36db2d18d89eff0345955aae979
+  patches:
+    - force-python-dynamic-link.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    - qt
+  missing_dso_whitelist:  
+    - "*/libnet_tracer.so.0"
+    - "*/liblefdef.so.0"
+    - "*/libpcb.so.0"
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ compiler('c') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxau') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('expat') }}  # [linux]
+    - {{ cdt('libpng') }}  # [linux]
```

Comment:
Typically, cdt packages are low-level and security libraries which we do not distribute for security or system-compatability reasons. I'm pretty sure cdt packages are just placeholders, so they won't be installed by conda for the end user.

Please check our [registry of existing packages](https://conda-forge.org/feedstock-outputs/). If you find the package, it should probably be a non-cdt package added to the host section. For example, libpng probably doesn't need to be a cdt package.

https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl